### PR TITLE
Modified add/edit event modals + fixed bugs connected to them

### PR DESCRIPTION
**Pull request by christykobyletska**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by andrii52 — APPROVED** [review]

File: src/components/Notifications/Messages.tsx [hunk 1]
```diff
@@ -48,6 +48,10 @@ export const onlyWhiteSpaces = ()=>{
     return `Назва не може містити лише порожні символи`
 };
 
+export const inputOnlyWhiteSpaces = ()=>{
+    return `Поле не може містити лише порожні символи`
+};
```
Comment (Tali32) [inline src/components/Notifications/Messages.tsx:53]: Don't you think that the previous function at lines 47-49 is actually the same?

File: src/models/GllobalValidations/DescriptionValidation.tsx [hunk 2]
```diff
@@ -182,6 +181,35 @@ import{
     {
         required: true,
         message: emptyInput(),
-    }
-    
+    },
+    Inputs: 
+    [
+        {
+            pattern: /^\s*\S.*$/,
```
Comment (Tali32) [inline src/models/GllobalValidations/DescriptionValidation.tsx:188]: Can there be two or more spaces between words?

File: src/models/GllobalValidations/DescriptionValidation.tsx [hunk 3]
```diff
@@ -182,6 +181,35 @@ import{
...
+        },
+        {
+            required: true,
+            message: emptyInput(),
+        },    
+    ],
+    DescriptionAndQuestions:
+    [
+        {
+            pattern: /^\s*\S.*$/,
```
Comment (Tali32) [inline src/models/GllobalValidations/DescriptionValidation.tsx:203]: And there?

**Review by Tali32 — CHANGES_REQUESTED** [review]

File: src/models/GllobalValidations/DescriptionValidation.tsx (hunk 2 above)
Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:188]: yes

Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:188]: it only prevent entering just whitespaces, as was in requirements

File: src/models/GllobalValidations/DescriptionValidation.tsx (hunk 3 above)
Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:203]: the same

File: src/components/Notifications/Messages.tsx (hunk 1 above)
Comment (christykobyletska) [reply src/components/Notifications/Messages.tsx:53]: you are right, I can delete first one and use only second

Comment (christykobyletska) [reply src/components/Notifications/Messages.tsx]: Done

**Review by Tali32 — APPROVED** [review]
Nice!

**Review by Oleh-Bodnar — APPROVED** [review]

**Pull request merged**